raja/timing/timings_ray_2d_p1_ms10.org

```diff
@@ -0,0 +1,65 @@
+|H1order|refs|H1dofs|L2dofs|H1cg_rate|L2cg_rate|forces_rate|update_quad_rate|total_time|total_rate|
```

Comment:
Do we need to add these?

Reply:
I believe @camierjs is planning to clean this directory. For now I just added it as it was in the branch.